UnityProject/Assets/Scripts/Clothing/Eyewear/NightVisionGoggles.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using CameraEffects;
+
+public class NightVisionGoggles : MonoBehaviour, IServerInventoryMove
+{
+	[SerializeField, Tooltip("How far the player will be able to see in the dark while he has the goggles on.")]
+	private Vector3 nightVisionVisibility;
+
+	[SerializeField, Tooltip("The default minimal visibility size.")]
+	private Vector3 defaultVisionVisibility;
+
+
+	public void OnInventoryMoveServer(InventoryMove info)
+		{
+			RegisterPlayer registerPlayer;
```

Comment:
RegisterPlayer registerPlayer = info.ToRootPlayer;

Then you can get rid of the ones at line 21 and 31